maistra/run-ci.sh

```diff
@@ -16,13 +16,12 @@ export BUILD_SCM_REVISION="Maistra PR #${PULL_NUMBER:-undefined}"
 export BUILD_SCM_STATUS="SHA=${PULL_PULL_SHA:-undefined}"
 
 COMMON_FLAGS="\
-    --incompatible_linkopts_to_linklibs \
+    --config=clang \
```

Comment:
I wonder, can we do without this ` --config=clang`?

Reply:
Apparently there's a bazel build that does not identify correctly the compiler, which makes bazel/envoy think it's gcc and then enabling gcc-specific options. Then we get *lots* of warnings about unknown flags in clang. This basically declares the variable `BAZEL_COMPILER=clang`

Reply:
Aaah I see, sounds like there's good a good reason for it to be there 👍